tests/p2p/test_server.py

```diff
@@ -44,8 +44,15 @@ def get_open_port():
 INITIATOR_REMOTE = Node(INITIATOR_PUBKEY, INITIATOR_ADDRESS)
 
 
-class MockPeerPool:
+class MockPeerPool():
     is_full = False
+    connected_nodes = {}
```

Comment:
I think we should have test for this or maybe at least open an issue to not forget about them.